src/fileActions.jsx

```diff
@@ -454,12 +452,10 @@ export const EditPermissionsModal = ({ selected, path }) => {
     return (
         <Modal
           position="top"
-          variant="medium"
-          title={selected.type === "file"
-              ? _("File properties and access")
-              : selected.type === "link"
-                  ? _("Link properties and access")
-                  : _("Directory properties and access")}
+          variant={ModalVariant.small}
+          /* Translators: $0 represents a filename */
```

Comment:
Hah, we don't use this too often, maybe we should more often.

Reply:
But this can be a directory as well right?

Reply:
Yes.  This is a GNOME thing.

Reply:
Hahaha, noticed this in GNOME control panel yeah :)